coverage-tests.js

```diff
@@ -1300,22 +1310,22 @@ test('should send dom and location when check region by selector fully with cust
   },
   test({driver, eyes, assert, helpers}) {
     eyes.open({appName: 'Applitools Eyes SDK', viewportSize})
-    driver.executeScript('window.scrollTo(0, 350)')
+    driver.executeScript('window.scrollTo(0, 350)') // not really needed
```

Comment:
You can remove all of this. Since image location doesn't relate to scrolling anymore. Before it makes sense to do something like this to ensure that the image position is not affected by the scroll position.